package.json

```diff
@@ -57,7 +57,7 @@
     "passport-twitter": "^1.0.2",
     "phantomjs": ">=1.9.0",
     "serve-favicon": "^2.3.0",
-    "socket.io": "^1.3.5",
+    "socket.io": "Automattic/socket.io",
```

Comment:
@codydaig I was talking to @mleanos about that. What we need would be the development version of socket.io-client for testing, since it has the `extraHeaders` included. This is the only reasonable way to test socket.io at the moment. Other ways include varieties of hacks, that are not worth including. I'll give this a shot in about 4-5 hours (when it's evening time and I'm off work).